bridge/scripts/mint-script.ts

```diff
@@ -0,0 +1,300 @@
+        senderSignature: senderSignature.data,
+    });
+    await safeService.proposeTransaction({
+        safeAddress: checkSummedSafeAddress,
+        safeTransactionData: safeTransaction.data,
+        safeTxHash,
+        senderAddress: checkSummedSenderAddress,
+        senderSignature: senderSignature.data,
+    });
+
+    return safeTxHash;
+}
+
+async function confirmTransaction(safeTxHash: string) {
+    console.log("confirmTransaction", safeTxHash);
+
+    const ethAdapterOwner2 = new EthersAdapter({
+        ethers,
+        signerOrProvider: owner2Signer,
+    });
+
+    const safeSdkOwner2 = await Safe.create({
+        ethAdapter: ethAdapterOwner2,
+        safeAddress,
+    });
+
+    const signature = await safeSdkOwner2.signTransactionHash(safeTxHash);
+    const response = await safeService.confirmTransaction(
+        safeTxHash,
+        signature.data
+    );
+
+    console.log("Transaction confirmed:", response);
+    return { safeTxHash, confirmationResponse: response };
+}
+
+async function executeTransaction(
+    safeTxHash: string,
+    safeSdk: Safe = safeSdkOwner1
+) {
+    let safeBalance = await safeSdk.getBalance();
+
+    console.log(
+        `[Before Transaction] Safe Balance: ${ethers.utils.formatUnits(
+            safeBalance,
+            "ether"
+        )} ETH`
+    );
+
+    const safeTransaction = await safeService.getTransaction(safeTxHash);
+    const executeTxResponse = await safeSdk.executeTransaction(safeTransaction);
+    const receipt = await executeTxResponse.transactionResponse?.wait();
+
+    console.log("Transaction executed:");
+    console.log(
+        `${process.env.ETHERSCAN_ROOT_URL}/tx/${receipt?.transactionHash}`
+    );
+
+    safeBalance = await safeSdk.getBalance();
+
+    console.log(
+        `[After Transaction] Safe Balance: ${ethers.utils.formatUnits(
+            safeBalance,
+            "ether"
+        )} ETH`
+    );
+}
+
+function sleep(sec: number) {
+    return new Promise((resolve) => setTimeout(resolve, sec * 1000));
+}
+
+async function main(destination: string, amount: string, gasPrice: string) {
+    const amountToNum = Number(amount);
+    const gasPriceToNum = Number(gasPrice);
+
+    if (Number.isNaN(amountToNum)) {
+        console.log(`amount:${amount} is not a number`);
+        console.log("Cancel Minting...");
+        process.exit(1);
+    }
+
+    if (Number.isNaN(gasPriceToNum)) {
+        console.log(`gasPrice:${gasPrice} is not a number`);
+        console.log("Cancel Minting...");
+        process.exit(1);
+    }
+
+    // Validation Check of Transfer Amount
+    if (
+        amountToNum > Number(process.env.MAX_TRANSFER_AMOUNT) ||
+        amountToNum === 0
+    ) {
+        console.log("Invalid Amount Input");
+        console.log("Cancel Minting...");
+        process.exit(1);
+    }
+
+    try {
+        console.log("destination??", destination);
+        ethers.utils.getAddress(destination);
+    } catch (e) {
+        console.error(e);
```

Comment:
It seems better to let users know the error was shown while what context.

```suggestion
        console.error("While checking the destination address, error occurred: ", e);
```